Formula/man-db.rb

```diff
@@ -19,7 +19,7 @@ class ManDb < Formula
 
   depends_on "pkg-config" => :build
 
-  uses_from_macos "groff"
+  depends_on "groff"
```

Comment:
Most other formula put their `depends_on` statements together, so this should probably go right below `depends_on "pkg-config" => :build`, with a space before `uses_from_macos`.

Reply:
@tyrak, please run `brew audit --strict --online --fix man-db`, then push any changes made. `--fix` autocorrects what it can, so you don't have to.

Reply:
I fixed this one.